### PR TITLE
itvr-404-417-405

### PR DESCRIPTION
**Pull request by tim738745**

### CODE REVIEW

File: django/api/models/go_electric_rebate_application.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ class Status(TextChoices):
     )
     date_of_birth = DateField(validators=[validate_driving_age], null=True)
     tax_year = IntegerField(null=True)
+    approval_email_sent = BooleanField(null=True)
```
Comment (AlexZorkin) [inline django/api/models/go_electric_rebate_application.py:84]: What's the reasoning behind having a null boolean option here?

File: django/api/scheduled_jobs.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ def schedule_send_rebates_to_ncda():
             name="send_rebates_to_ncda",
             schedule_type="C",
             cron="15 * * * *",
+            q_options={"timeout": 1200, "ack_failure": True},
```
Comment (AlexZorkin) [inline django/api/scheduled_jobs.py:15]: Do we want to track failed tasks as successful using ack_failure: True? I could use a little background on the precedent with this one.

File: django/api/settings.py [hunk 3]
```diff
@@ -198,7 +198,7 @@
     "name": "ITVR",
     "workers": 4,
     "timeout": 90,
-    "retry": 120,
+    "retry": 1260,
```
Comment (AlexZorkin) [inline django/api/settings.py:201]: is 21 minutes used here to retry after the 1200 second timeout in the scheduled job?

File: django/api/models/go_electric_rebate_application.py (hunk 1 above)
Comment (tim738745) [reply django/api/models/go_electric_rebate_application.py:84]: My thinking was that there are some states an application can be in where it may not make sense for this field to be either true or false; for example, states of "household_initiated", "submitted", "verified" are not yet at the approval/non-approval stage.

File: django/api/scheduled_jobs.py (hunk 2 above)
Comment (tim738745) [reply django/api/scheduled_jobs.py:15]: For this particular job, I think it makes sense because, by default, ack_failure=False, and when that is the case, the task will keep on retrying until it succeeds and we want to avoid that with this job because of the ncda throttling issue.

File: django/api/settings.py (hunk 3 above)
Comment (tim738745) [reply django/api/settings.py:201]: So from the django_q docs, the retry time is "The number of seconds a broker will wait for a cluster to finish a task, before it’s presented again." Also: "The value must be bigger than the time it takes to complete longest task, i.e. [timeout](https://django-q.readthedocs.io/en/latest/configure.html#timeout) must be less than retry value and all tasks must complete in less time than the selected retry time."

Basically, the retry time has to be longer than the 1200 second timeout of the job because if, for example, the retry time is 600 seconds and the job takes 900 seconds, then at the 600 second mark (even though the job is still running and has not yet timed out), django_q will execute another instance of this job.

**Review by AlexZorkin — APPROVED** [review]
